ui-modules/blueprint-composer/app/views/main/graphical/edit/dsl/edit.dsl.controller.js

```diff
@@ -131,27 +127,31 @@ export const graphicalEditDslState = {
         index: {
             value: '',
             squash: true
-        }
+        },
+        isConfig: true // This flag identifies whether DSL edit is for configuration or something else. Configuration is a default.
```

Comment:
This flag is not used in Brooklyn code just now, however, branding parts rely on it.

Reply:
would be nicer to call this something like `subsection` which might take values `config`, `location`, etc

Reply:
Changed to subsection.